[Enhancement]: Minetest 5.8.0

### What changes would you like?

*Bonjour.*

https://dev.minetest.net/Changelog#5.7.0_.E2.86.92_5.8.0

IrlrichtMt13

### Any extra information?

_No response_

Sadly, it fails to build:

```
-- Found IrrlichtMt 1.9.0.10
CMake Error at CMakeLists.txt:148 (message):
  At least IrrlichtMt 1.9.0mt13 is required to build
```

Full log: https://launchpad.net/~build.snapcraft.io/+snap/0fbc1d1142be3edd01556d4ec3766968/+build/2317610

--- a/snap/snapcraft.yaml
+++ b/snap/snapcraft.yaml
@@ -38,7 +38,7 @@ parts:
 
   irrlicht:
     source: https://github.com/minetest/irrlicht.git
-    source-tag: 1.9.0mt10 # must be updated to match whatever a specific minetest version requires
+    source-tag: 1.9.0mt13 # must be updated to match whatever a specific minetest version requires
     plugin: cmake
     configflags:
       - -DBUILD_SHARED_LIBS=OFF

